Week1/databases/Homework/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Usually it's a good idea to add a .md file with instructions on how to use your project, this file usually has:

- dependencies, maybe how to install them
- how to use your code ( how to run it, if you have provide an API, how to use, which parameters are required, etc)
- etc